bulk-print-fail-alert.tf

```diff
@@ -0,0 +1,18 @@
+module "cmc-bulk-print-fail-alert" {
+  source = "git@github.com:hmcts/cnp-module-metric-alert"
+  location = "${(var.env == "prod") ? var.location : "West Europe"}"
```

Comment:
why is there a different location for prod?

Reply:
for prod I'm saying use whatever location is, but the default (UK South) isn't valid for an env of sandbox; and neither is the only location available for sandbox action groups, which is global. It was quite irritating.

Reply:
it'll be the exact same for prod, app insights is available in west europe and not many other places

Reply:
So is there a change you want me to make? This complexity is purely to make it work in sandbox and test environments

Reply:
its a change you have to make as it won't work as is in prod